Dockerfile

```diff
@@ -30,7 +30,6 @@ ARG ARCH
 ARG LLVM_VERSION
 ARG CXX_COMMON_VERSION
 ARG LIBRARIES
-ARG REMILL_COMMIT_ID=master
```

Comment:
This argument is harder to support with submodules and it didn't seem very important to me so I just removed it. If someone wants to build with a different commit, they can just modify the submodule.